src/Kestrel.Transport.Sockets/Internal/ISocketTrace.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal
+{
+    public interface ISocketTrace : ILogger
```

Comment:
I think this should be called ISocket**s**Trace, given the transport is called Socket**s**.